x-pack/legacy/plugins/searchprofiler/public/app_util.js

```diff
@@ -4,12 +4,19 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+// Convert triple quotes into regular quotes and escape internal quotes.
+function collapseLiteralStrings(data) {
```

Comment:
This function is copy-pasted from Console: https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/console/public/src/utils.js#L61